l3kernel/l3tl.dtx

```diff
@@ -2213,6 +2229,23 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}{\tl_rescan_dynamic:n, \tl_rescan_dynamic:V}
+%   A thin wrapper from \tn{scantokens}. At present, we are told by
+%   Hans that LuaMeta\TeX{} will retain this primitive, so we should be
+%   OK for all engines.
+%    \begin{macrocode}
+\cs_gset_protected:Npn \tl_rescan_dynamic:n #1
```

Comment:
Should be `new` then.
```suggestion
\cs_set_protected:Npn \tl_rescan_dynamic:n #1
```

Reply:
This is deliberate - as we are wanting a global def (yes, i know we should be at top-level but if you look at `new` it works globally).